README.md

```diff
@@ -1,70 +1,242 @@
+
+### SSL Setup
+
+To simplify the setup only the ELB (Public Side) is setup with SSL.
+
+Learn how to upload certificates to be used with this [Guide](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_server-certs.html)
+
+To Get an existing SSL Cert in AWS Follow this [Guide](http://docs.aws.amazon.com/cli/latest/reference/iam/get-server-certificate.html)
+
+or run this command `aws iam get-server-certificate --server-certificate-name`
+
+You can also use the new [AWS Certificate Manager Service](http://docs.aws.amazon.com/acm/latest/userguide/gs-acm.html) to create and manage certificates
+
+### Blue/Green Deployment
+
+This setup supports doing staged deploys and/or blue/green deployments
+
+Below describes how this is accomplished, but there is a manual step require to complete the deploy.
+
+-   Manual step
+    -   Change `chef.$HostedZone` in Route53 to/from `chef-a.$HostedZone` or `chef-b.$HostedZone`
+
+
+-   Blue/Green Options:
+    -   ChefSubdomain Param; Enabled having a chef-a, chef-b, and even a chef-test subdomain created
+        -   Subnets are changed based on these settings
+    -   Server names are created based on the Subdomain and a prime chef.domain.com to support the multi-dns setup; [Code](aws_backendless_chef_ha.yml#L1149)
+    -   There is a workaround for the odd cert creation caused by the multiple server name setup;
+        -   Specify Cert Location: [Code 01](aws_backendless_chef_ha.yml#L1147-L1148)
+        -   Copy badly name cert to expected one: [Code 02](aws_backendless_chef_ha.yml#L1194-L1200)
+
+**In place upgrades are not supported yet.**
+
+#### Subnet Switching
+
+To support using the blue/green domains. There is a switch in place depending on if you select chef-a or chef-b
+-   Subnet Section [Code 01](aws_backendless_chef_ha.yml#L357-L408)
+-   Switch Command [Code 02](https://github.com/HearstAT/cfn_backendless_chef/blob/cfn_yml/aws_backendless_chef_ha.yml#L368)
+-   Conditional [Code 03](aws_backendless_chef_ha.yml#L336-L337)
+
+Chef-a has it's own set of subnets.
+
+chef-b and chef-test chare subnets, as the intention for chef-test is to test your deploys/changes/etc in another region/vpc
+
+### Restore/Backup Options
+
+#### Database Snapshot
+
+This is the simplest option from a all AWS standpoint. You also keep the same pivotal credentials
+
+-   Required Items
+    -   Existing S3 Bucket with:
+        -   etc_opscode directory
+        -   etc_reporting directory (can be empty)
+        -   migration-level
+    -   DB Snapshot (Same Region)
+-   Required Params
+    -   ExistingBucketName (e.g.; chef-data-bucket)
+    -   DBSnapShot (e.g.; snaphot-date)
+
+#### Knife EC Restore
+If looking to change architecture, DB credentials, or even Pivotal then this is the best option.
+
+This can be down outside the entire Cloudformation process, but if wanting to do it inline see below.
+
+-   Required Items
+    -   [knife ec](https://github.com/chef/knife-ec-backup) backup in a tar file
```

Comment:
move the "backup" into the link `knife ec backup` as opposed to `knife ec` backup?